universal-login-relayer/lib/integration/sql/services/PendingMessagesSQLStore.ts

```diff
@@ -63,12 +70,20 @@ export class PendingMessagesSQLStore implements IPendingMessagesStore {
     const message = await this.get(messageHash);
```

Comment:
getStatus -> extract into 2 methods:
`core/MessageService.getStatus()` that should use: 
`int/ethereum/WalletContract.getRequiredSignatures()` an
`int/sql/PendingMessageSQLStore.get()`